test/JDBC/expected/babel-4517-vu-verify.out

```diff
@@ -0,0 +1,107 @@
+BEGIN TRANSACTION babel_4517
+GO
+
+SELECT set_config('babelfishpg_tsql.explain_costs', 'off', true)
+GO
+~~START~~
+text
+off
+~~END~~
+
+
+SELECT set_config('debug_parallel_query', '0', true)
```

Comment:
Is this necessary? If so,
1. please reset this at the end of this file.
2. you need to rename this to `force_parallel_mode` when downporting to lower major versions

Reply:
Yes, it's for parallel query regression test, to turn  parallel query off in default, this value will be reset back after the transaction is committed ( transaction : babel_4517 ), so that's why set it with set_config('debug_parallel_query', '0', true) -> true .

Yes, I knew I'll use force_parallel_mode.

Reply:
I can't direct turn on/off after the test case, due to this value is not a fixed value compared to jdbc test and parallel query test